src/test/java/rudi/process/program/ProgramLineProcessorTests.java

```diff
@@ -0,0 +1,33 @@
+package rudi.process.program;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import rudi.process.LineProcessor;
+import rudi.support.RudiSource;
+import rudi.support.RudiStack;
+
+import java.util.Arrays;
+
+@RunWith(JUnit4.class)
+public class ProgramLineProcessorTests {
+
+    @Before
+    public void setup() {
+        RudiStack.getInstance().removeAllElements();
```

Comment:
Without this, there was one thing in the stack when it started. I can't figure out why.